client/src/Movies/MovieCard.js

```diff
@@ -1,7 +1,67 @@
 import React from 'react';
+import axios from 'axios';
 
-const MovieCard = props => {
-  return;
-};
+
+class MovieCard extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      movies: []
+    };
+  }
+
+  componentDidMount() { 
+
+    const id = this.props.match.params.id;
```

Comment:
This logic is not necessarily reusable - if you're expecting there to be a match.params, then this component could only be rendered through a Route. The idea behind MovieCard is to reuse code that you're using in Movie and MovieDetails